samples/drivers/jesd216/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&mx25uw63 {
```

Comment:
The order of commits is wrong. The above node label is added in the third commit but it is referred in the first one.

Reply:
nit: also "soc: nrf54h: Enable SPI DW HSSI register layout" should be before "boards: nordic: add mx25uw6345g flash node to nRF54H20dk" that activates the spi_dw driver.

Reply:
Done